src/govuk/components/footer/footer.yaml

```diff
@@ -67,6 +67,22 @@ params:
       type: object
       required: false
       description: HTML attributes (for example data attributes) to add to the anchor in the footer navigation section.
+- name: contentLicence
+  type: string
+  required: false
+  description: HTML or text to replace the default Open Government Licence notice with.
+- name: hideOpenGovernmentLicenceLogo
+  type: boolean
+  required: false
+  description: Removes the Open Government Licence logo, if the licence being used isn't the OGL.
+- name: copyright
+  type: string
+  required: false
+  description: Text to replace the default Crown Copyright notice with.
+- name: hideCrownCopyrightArms
+  type: boolean
+  required: false
+  description: Removes the Crown Copyright notice and royal coat of arms.
```

Comment:
Would be good to clarify what will be required and suggested in this case. Does the Design System/GDS care if users don't add a new copyright/license to replace these?

Reply:
I'm not sure how necessary that is, might be a question for policy? 

For public-facing GOV.UK services, the default (or a translation of the default) is expected, I imagine. 

For internal/caseworking systems, they're more likely to remove or change the content licence and potentially remove the coat of arms for space purposes. 

For third-parties that use Frontend as a boilerplate, they'll probably remove or replace all of it.